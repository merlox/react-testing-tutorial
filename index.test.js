import React from 'react'
import renderer from 'react-test-renderer'
import App from './index'

test('App loads', () => {
  const component = renderer.create(
    <App />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
