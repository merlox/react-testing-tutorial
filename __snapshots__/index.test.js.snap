// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App loads 1`] = `
<div>
  Working
</div>
`;
